### DISCUSSION TITLE
Todo for bevy 1.0?

### DISCUSSION THREAD

**Discussion by Fidius-jko** [bevyengine/bevy#14530] [Ideas]
What is the minimum the community should do for bevy 1.0?

**Comment by bugsweeper** [comment]
It's very much relative. Many people think that 1.0 is about production ready. What should bevy have, to be good enought relative to other game engine? Should it have scene editor (and UI system for it)? Should it have easy export/upload to main platforms? Should it have rich template library for quick start? For 1.0 usualy crate should stabilize it's API. What community should do for API stabilization? I think API should be usefull, community could try to use and give feedback about usefullness. Also community could try other game engines and propose best sides of them as improvement (maybe in discussions?).
